src/main/resources/assets/base/models/item/cluster.json

```diff
@@ -0,0 +1,7 @@
+{

```

Comment:
This file isn't needed

Reply:
Not sure how this got in.. will remove it

Reply:
what would I use instead?